src/Service.ts

```diff
@@ -38,7 +36,7 @@ export default class Service {
       | Routes.datasets._account._dataset.services._serviceName.Get
     >({
       errorWithCleanerStack: getErr(
-        `Failed to get information of service ${this._name} in dataset ${this._datasetNameWithOwner}.`
+        `Failed to get information of service '${this._name}' in dataset '${(await this._dataset.getInfo(true)).name}'.`
```

Comment:
Do we need `await this._dataset.getInfo(true)`? That force-refreshes the info afaik. Cant we do `await this._dataset.getInfo()`?